src/main/scala/mesosphere/chaos/http/impl/HttpServer.scala

```diff
@@ -151,15 +161,6 @@ class HttpServer(conf: HttpConf, registry: MetricRegistry) extends ServletServer
 
   private lazy val requestLog: RequestLog = new ChaosRequestLog
 
-  private lazy val resourceHandler: ResourceHandler = {
```

Comment:
@aquamatthias out of curiosity - why is there now need for a ResourceHandler anymore?